explorer/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "datomish-explorer"
+version = "0.0.1"
+
+[dependencies]
+iron = "0.4.x"
```

Comment:
any reason to use 0.4.x here but an explicit version for the others?  not really sure best practice here